### PR TITLE
x11-misc/copyq: 

### PR DESCRIPTION
**Pull request by truffle0**
---

Please check all the boxes that apply:

- [ ] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [ ] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [ ] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [ ] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by thesamesam** [comment]
? You can ping if things are stuck.

**Comment by truffle0** [comment]
i've just been cleaning up a few things, didn't think there was much point keeping this open when the upstream version is up to 13.0.0 now

File: x11-misc/copyq/copyq-10.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	X? ( x11-base/xorg-proto )
+"
+BDEPEND="
+	kde-frameworks/extra-cmake-modules:0
+	dev-qt/qttools:6[linguist]
+	dev-util/wayland-scanner
+	test? (
+		app-crypt/gnupg
+		x11-wm/openbox
+		x11-base/xorg-server[xvfb]
```
Comment (a17r) [inline x11-misc/copyq/copyq-10.0.0.ebuild:47]: already provided for by `virtualx.eclass`

File: x11-misc/copyq/copyq-10.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	sleep 5
+	eend 0
+
+	"${BUILD_DIR}"/copyq tests
+}
+
+src_test() {
+	ewarn "Many tests are very sensitive to specific timing/behaviour in the QT"
+	ewarn "library. It's not uncommon to have a few edge-case tests fail due to"
+	ewarn "this, and the application will likely still run without issue."
```
Comment (a17r) [inline x11-misc/copyq/copyq-10.0.0.ebuild:99]: I didn't think this was worth pointing out [it could be said for any GUI tests], so I didn't include it in 13.0.0 bump.
